test/built-ins/Function/prototype/toString/bound-function.js

```diff
@@ -1,12 +1,17 @@
 // Copyright (C) 2016 Michael Ficarra. All rights reserved.
+// Copyright (C) 2018 the V8 project authors. All rights reserved.
 // This code is governed by the BSD license found in the LICENSE file.
-
 /*---
 esid: sec-function.prototype.tostring
-description: Function.prototype.toString on bound function exotic objects
+description: >
+  toString of Proxy for function target does not throw (bound Function Expression)
```

Comment:
is this really Proxy?

Reply:
you already answered this... please ignore. 

It seems wrong, we might wanna fix it later